### PR TITLE
Eventing core 0.19.2

### PR DESCRIPTION
**Pull request by matzew**
This PR is made out of a few commits (details on the commit message):

* set version on `project.yaml`
* patch the patch (LOL)
* generate the file via `make`

### CODE REVIEW

**Comment by matzew** [comment]
/hold
let's see how this goes :smile:

**Comment by matzew** [comment]
> `test_runner.go:160: Couldn't initialize clients: error while retrieving the config-tracing config map: configmaps "config-tracing" not found`

checking...

File: go.mod [hunk 1]
```diff
@@ -7,28 +7,26 @@ require (
...
 	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/pflag v1.0.5
-	go.uber.org/zap v1.15.0
+	go.uber.org/zap v1.16.0
 	google.golang.org/genproto v0.0.0-20200914193844-75d14daec038 // indirect
 	k8s.io/api v0.19.2
 	k8s.io/apimachinery v0.19.2
 	k8s.io/client-go v12.0.0+incompatible
-	knative.dev/eventing v0.18.5-0.20201105155307-650096a39064
+	knative.dev/eventing v0.19.2
```
Comment (markusthoemmes) [inline go.mod:25]: This should not be done via go.mod but **has** to be done via updating `hack/update-deps.sh`. We should probably add a check to the actions that prevent manual go.mod mucking for individual projects that should be bumped as a whole.

**Comment by matzew** [comment]
@cardil  looking here:

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-knative_serverless-operator/745/pull-ci-openshift-knative-serverless-operator-master-4.6-upgrade-tests-aws-ocp-46/1336945790995664896#1:build-log.txt%3A1232

is there something we need to fix for the upgrade tests?  and the system namespace ?

**Comment by aliok** [comment]
/test 4.6-upgrade-tests-aws-ocp-46

```
=== FAIL: test/upgrade TestPreUpgrade/InMemoryChannel-messaging.knative.dev/v1beta1 (0.34s)
    channel_single_event_helper.go:62: Run test with channel {"InMemoryChannel" "messaging.knative.dev/v1beta1"}
    test_runner.go:153: namespace is : "test-pre-upgrade-in-memory-channel-messaging-knative-dev-vh6gqn"
    test_runner.go:160: Couldn't initialize clients: error while retrieving the config-tracing config map: configmaps "config-tracing" not found
```

Error doesn't make sense. I see no changes around `config-tracing` in our manifest.

**Comment by aliok** [comment]
/lgtm
/approve

looks good. let's merge if build is green.

**Comment by matzew** [comment]
```

=== FAIL: test/upgrade TestServicePreUpgradeAndScaleToZero (32.51s)
    service.go:113: Creating a new Service service scale-to-zero-upgrade-service
    crd.go:35:  resource {<nil> <nil> <*>{&TypeMeta{Kind:,APIVersion:,} &ObjectMeta{Name:scale-to-zero-upgrade-service,GenerateName:,Namespace:,SelfLink:,UID:,ResourceVersion:,Generation:0,CreationTimestamp:0001-01-01 00:00:00 +0000 UTC,DeletionTimestamp:<nil>,DeletionGracePeriodSeconds:nil,Labels:map[string]string{},Annotations:map[string]string{},OwnerReferences:[]OwnerReference{},Finalizers:[],ClusterName:,ManagedFields:[]ManagedFieldsEntry{},} {{&ObjectMeta{Name:,GenerateName:,Namespace:,SelfLink:,UID:,ResourceVersion:,Generation:0,CreationTimestamp:0001-01-01 00:00:00 +0000 UTC,DeletionTimestamp:<nil>,DeletionGracePeriodSeconds:nil,Labels:map[string]string{},Annotations:map[string]string{autoscaling.knative.dev/window: 6s,},OwnerReferences:[]OwnerReference{},Finalizers:[],ClusterName:,ManagedFields:[]ManagedFieldsEntry{},}} {<nil>}} {{0 <nil> <nil>} { } {<nil> <nil> <nil>}}} <nil>}
    service.go:131: Waiting for Service to transition to Ready. service scale-to-zero-upgrade-service
    service.go:136: Checking to ensure Service Status is populated for Ready service
    service.go:142: Getting latest objects Created by Service
skipped 6 lines unfold_more
DONE 4 tests, 2 failures in 63.229s
Finished run, return code is 1
XML report written to /tmp/artifacts/build-jgPdWsHE/junit_X5msUu85.xml
JUnit file /tmp/artifacts/build-jgPdWsHE/junit_E6mE1iIX.xml is created for reporting the test result
2020/12/11 16:28:53 Getting artifacts dir from prow
16:28:53.996 ERROR:   🚨 Error (code: 1) occurred at ./test/lib.bash:383, with command: return ${failed}
16:28:53.999 INFO:    Dumping state...
16:28:54.001 DEBUG:   Environment variables:
PROW_JOB_ID=9e2c0bfd-3bc4-11eb-a5be-0a580a830181
TEST_KNATIVE_E2E=false
ARTIFACTS=/tmp/artifacts/build-jgPdWsHE
skipped 261 lines unfold_more
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
16:28:55.231 INFO:    Gather knative state
16:30:21.566 ERROR:   Stack trace:
16:30:21.568 ERROR:    1: /go/src/github.com/openshift-knative/serverless-operator/test/serving.bash:124 go_test_e2e(...)
16:30:21.570 ERROR:    2: ./test/lib.bash:187 run_serving_preupgrade_test(...)
16:30:21.572 ERROR:    3: ./test/upstream-e2e-tests.sh:23 run_rolling_upgrade_tests(...)
```



/test 4.6-upgrade-tests-aws-ocp-46

@nak3 @markusthoemmes   seen the above before?  Or ... does it mean we need to bump serving.eventing image in one? Like we used to do ?

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -319,29 +319,29 @@ spec:
...
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-controller"
                       - name: "IMAGE_sugar-controller__controller"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-sugar-controller"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-sugar-controller"
                       - name: "IMAGE_eventing-webhook__eventing-webhook"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-webhook"
-                      - name: "IMAGE_storage-version-migration-eventing-eventing-0.18.6__migrate"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-storage-version-migration"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-webhook"
+                      - name: "IMAGE_storage-version-migration-eventing-eventing-0.19.2__migrate"
```
Comment (lberk) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:327]: Do we still need the migrate image? (we don't have a pre-install job anymore do we?)

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 3]
```diff
@@ -432,29 +432,29 @@ spec:
...
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-storage-version-migration"
                       - name: "IMAGE_mt-broker-controller__mt-broker-controller"
-                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-mtchannel-broker"
+                        value: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-mtchannel-broker"
```
Comment (lberk) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:443]: same here with the migrate image

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml [hunk 4]
```diff
@@ -705,29 +705,29 @@ spec:
...
+      image: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-controller"
     - name: "IMAGE_sugar-controller__controller"
-      image: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-sugar-controller"
+      image: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-sugar-controller"
     - name: "IMAGE_eventing-webhook__eventing-webhook"
-      image: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-webhook"
-    - name: "IMAGE_storage-version-migration-eventing-eventing-0.18.6__migrate"
-      image: "registry.svc.ci.openshift.org/openshift/knative-v0.18.6:knative-eventing-storage-version-migration"
+      image: "registry.svc.ci.openshift.org/openshift/knative-v0.19.2:knative-eventing-webhook"
+    - name: "IMAGE_storage-version-migration-eventing-eventing-0.19.2__migrate"
```
Comment (lberk) [inline olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:713]: same here with the migrate image

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 2 above)
Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:327]: yes, we still build it

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 3 above)
Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:443]: same :-)

File: olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml (hunk 4 above)
Comment (matzew) [reply olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml:713]: same :-)

**Comment by lberk** [comment]
from an eventing perspective this looks good to me (but given the serving failure I'll reserve that for those who've been pinged)

**Comment by matzew** [comment]
```
error: build error: failed to pull image: After retrying 2 times, Pull image still failed due to error: unable to retrieve auth token: invalid username/password: unauthorized: unable to validate token: Timeout
```

/retest

**Pull request merged**